Remotion/ObjectBinding/Web/UI/Controls/BocRendererBase.cs

```diff
@@ -94,7 +95,7 @@ protected override void AddDiagnosticMetadataAttributes (RenderingContext<TContr
 
       var control = renderingContext.Control;
       if (!control.DisplayName.IsEmpty)
-        renderingContext.Writer.AddAttribute(DiagnosticMetadataAttributesForObjectBinding.DisplayName, control.DisplayName.ToString(WebStringEncoding.Attribute));
+        HtmlUtility.ExtractPlainText(control.DisplayName).AddAttributeTo(renderingContext.Writer, DiagnosticMetadataAttributesForObjectBinding.DisplayName);
```

Comment:
I think this is a bug similiar to RM-8443. Please add a test, check if this was broken before and if so, create a bug ticket. You could fix this as a separate branch (cleaner) or in here with a separate ticket number for this commit (cheaper).